design/categories.go

```diff
@@ -0,0 +1,51 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var category = a.Type("categories", func() {
```

Comment:
We don't need to expose categories as a controller.. these changes should not be needed.